opentreemap/treemap/lib/user.py

```diff
@@ -12,6 +12,14 @@
 from treemap.util import get_filterable_audit_models
 
 
+def _instance_ids_edited_by(user):
+    edited_instance_ids = Audit.objects.filter(user=user)\
+                                       .values_list('instance_id', flat=True)\
+                                       .distinct()
+    # In some cases a None is returned in the values list, so filter it out
+    return [x for x in edited_instance_ids if x]
```

Comment:
I think it's generally more idiomatic and would guess in this case more performant if this stays lazy as a queryset instead of forcing to a list. Why not `.exclude(instance_id=None)`? That way, callers can execute a single query.